test/server/Server.test.js

```diff
@@ -140,12 +139,12 @@ describe('Server', () => {
 
   describe('Invalidate Callback', () => {
     describe('Testing callback functions on calling invalidate without callback', () => {
-      it('should be `noop` (the default callback function)', (done) => {
+      it('should use default `noop` callback', (done) => {
         const compiler = webpack(config);
         const server = new Server(compiler, baseDevConfig);
 
         server.invalidate();
-        expect(server.middleware.context.callbacks[0]).toBe(noop);
+        expect(server.middleware.context.callbacks.length).toEqual(1);
```

Comment:
webpack-dev-middleware no longer exports the noop function. Should we make it get exported from webpack-dev-middleware, or do you think this is sufficient for this test?

Reply:
I think it's ok.